integration-tests/tests/redis/redis_metrics_test.go

```diff
@@ -0,0 +1,17 @@
+//go:build !windows
+
+package main
+
+import (
+	"testing"
+
+	"github.com/grafana/agent/integration-tests/common"
+)
+
+func TestRedisMetrics(t *testing.T) {
+	var redisMetrics = []string{
+		"redis_up",
+		"redis_memory_used_bytes",
```

Comment:
I wonder if we should start documenting the metrics which exporters create? We could add a new section to the component docs, or could mention them in the description.
Given that we have tests to prove that they work, it should be especially easy.
cc @wildum

Reply:
Similarly, maybe an idea here would be in the future to ingest this from a metrics file rather than have it hardcoded, similar to what we now export for the cloud integrations?